Sources/MapboxCoreNavigation/ActiveNavigationEventDetails.swift

```diff
@@ -57,7 +57,7 @@ struct ActiveNavigationEventDetails: NavigationEventDetails {
     init(dataSource: ActiveNavigationEventsManagerDataSource, session: SessionState, defaultInterface: Bool, appMetadata: [String: String?]? = nil) {
         coordinate = dataSource.router.rawLocation?.coordinate
         startTimestamp = session.departureTimestamp
-        sdkIdentifier = defaultInterface ? "mapbox-navigation-ui-ios" : "mapbox-navigation-ios"
+        sdkIdentifier = URLSession.navigationSdkIdentifierForTelemetry
```

Comment:
I guess this change should break all the monitorings using `userAgent` to distinguish iOS Nav SDK. 
Have we notified all possible users and updated the queries for our own monitorings?